app/models/queries/filters/shared/user_name_filter.rb

```diff
@@ -70,7 +70,7 @@ def sql_concat_name
       when :firstname_lastname
         "LOWER(CONCAT(users.firstname, CONCAT(' ', users.lastname)))"
       when :firstname
-        'LOWER(users.firstname)'
+        'LOWER(users.firstname))'
```

Comment:
what's that extra closing parenthesis for?

Reply:
User error, I Initially had the unaccent function in all these options